ansible/templates/config.hjson

```diff
@@ -21,4 +21,16 @@
     smtp_from_address: "noreply@{{ domain }}"
     use_tls: false
   }
+  # settings related to activitypub federation
+  federation: {
+    # whether to enable activitypub federation. 
+    enabled: true
```

Comment:
Please keep this disabled for now, I dont want people to have federation enabled without knowing it, especially now as we dont even enable it on our own prod instances. Plus this config will give open federation. If anything, it should be preconfigured to allow federation with a limited set of instances. But honestly its probably better to let people do this manually.

Reply:
Okay I can, it was just strange that when I did an ansible deploy of lemmy, ( https://fanonwave.ml ) I then had to SSH into the server and manually add this block to get it working.

Reply:
You could also edit the local lemmy.hjson to make any adjustments before first deploying it. Might be worth mentioning this in the docs. Also you can edit the config directly inside Lemmy, so I dont think this is a big problem.